ScarletEditor/Sources/EditorMain.cpp

```diff
@@ -7,44 +7,341 @@
+			{
+				Grid[x][y][z] = MakeSphereCollider(glm::vec3(x * scale - offset, y * scale + heightOffset, z * scale), glm::vec3(0.05f));
+			}
+		}
+	}
+
+	for (int x = 0; x < width; x++)
+	{
+		for (int y = 0; y < height; y++)
+		{
+			for (int z = 0; z < depth; z++)
+			{
+				std::vector<EID> Anchors;
+				std::vector<float> Lengths;
+
+				if (y < height - 1)
+				{
+					Anchors.push_back(Grid[x][y + 1][z]);
+					Lengths.push_back(scale);
+				}
+				if (x < width - 1)
+				{
+					Anchors.push_back(Grid[x + 1][y][z]);
+					Lengths.push_back(scale);
+				}
+				if (x < width - 1 && y < height - 1)
+				{
+					Anchors.push_back(Grid[x + 1][y + 1][z]);
+					Lengths.push_back(sqrt2);
+				}
+				if (z < depth - 1)
+				{
+					Anchors.push_back(Grid[x][y][z+1]);
+					Lengths.push_back(scale);
+				}
+				if (x < width - 1 && z < depth - 1)
+				{
+					Anchors.push_back(Grid[x + 1][y][z+1]);
+					Lengths.push_back(sqrt2);
+				}
+				if (x < width - 1 && y < height - 1 && z < depth - 1)
+				{
+					Anchors.push_back(Grid[x + 1][y + 1][z + 1]);
+					Lengths.push_back(sqrt3);
+				}
+
+				AddSpring(Grid[x][y][z], Anchors, Lengths, 1000.f);
+			}
+		}
+	}
+
+	MakeGround();
+}
+
+EID MakePlanet(glm::vec3 Pos, glm::vec3 V0, float Mass, float Scale, const char* Name)
+{
+	auto [Ent, Trans, Mesh, Rb] = GEditor->GetActiveWorld()->CreateEntity<Transform, StaticMeshComponent, RigidBodyComponent>(Name);
+
+	Trans->Position = Pos;
+	Trans->Rotation = glm::vec3(0.f, 0.f, 0.f);
+	Trans->Scale = glm::vec3(Scale * 0.05f);
+
+	Rb->Velocity = V0;
+	Rb->Mass = Mass;
+	Rb->UsesGravity = false;
+	Rb->UsesKeplerGravity = true;
+
+	MakeMesh(Mesh, "/ScarletEngine/Content/Sphere.obj");
+
+	return Ent->ID;
+};
+
+void DemoKepler()
+{
+	MakePlanet(glm::vec3(4.3f, -1.f, 0.f), glm::vec3(0.f, 0.f, 2.f), 1., 1.2f, "Jupiter");
+	MakePlanet(glm::vec3(3.f, 0.6f, 0.f), glm::vec3(0.f, 0.f, 2.5f), 1.f, 0.5f, "Venus");
+	MakePlanet(glm::vec3(4.15f, -0.4f, 0.f), glm::vec3(0., 0.f, 2.5f), 1.f, .3f, "Moon");
+	MakePlanet(glm::vec3(4.f, -0.4f, 0.f), glm::vec3(0.f, 0.f, 2.f), 7 * glm::pow(10, 8), 0.6f, "Earth");
+	MakePlanet(glm::vec3(2.f, 0.2f, 0.f), glm::vec3(0.f, 0.f, 2.5f), 1.f, 0.5f, "Mercury");
+	MakePlanet(glm::vec3(0.f, 0.f, 0.f), glm::vec3(0.f, 0.f, 0.f), 3 * glm::pow(10, 11), 2.f, "Sun");
+}
+
+#pragma endregion
+
 int main()
 {
-	using namespace ScarletEngine;
-	
 	// #todo_Core: this should be loaded by a config file or something, for now default it to this.
 	AssetManager::SetAssetRoot("../");
 
 	ModuleManager::GetInstance().RegisterModule<RenderModule>();
 	ModuleManager::GetInstance().RegisterModule<UIModule>();
 
+	SystemScheduler::Get().RegisterSystem<SpringSystem>();
+	SystemScheduler::Get().RegisterSystem<RigidBodySystem>();
+	SystemScheduler::Get().RegisterSystem<AABBvsAABBColliderSystem>();
+	SystemScheduler::Get().RegisterSystem<SphereVsSphereColliderSystem>();
+	SystemScheduler::Get().RegisterSystem<PlaneVsSphereColliderSystem>();
```

Comment:
In AchillesCore.h you should write a function `RegisterSystems()` and here you could just call `Achilles::RegisterSystems()` to make it a bit cleaner.